moped-database/views/uniform_features.sql

```diff
@@ -1,62 +1,98 @@
--- latest version 1678894870998_remove_cols_from_geo_views
+-- latest version from 1691698449987_add-district-columns-to-views
 DROP VIEW "public"."uniform_features";
 
-CREATE OR REPLACE VIEW "public"."uniform_features" AS
+CREATE VIEW "public"."uniform_features" AS
 SELECT
     feature_signals.id,
     feature_signals.component_id,
     'feature_signals'::text AS "table",
-    json_build_object(
-        'signal_id', feature_signals.signal_id, 'knack_id', feature_signals.knack_id, 'location_name', feature_signals.location_name, 'signal_type', feature_signals.signal_type) AS attributes,
-    feature_signals.geography
+    json_build_object('signal_id', feature_signals.signal_id, 'knack_id', feature_signals.knack_id, 'location_name', feature_signals.location_name, 'signal_type', feature_signals.signal_type) AS attributes,
+    feature_signals.geography,
+    districts.council_districts
 FROM
     feature_signals
-WHERE (
-    feature_signals.is_deleted = FALSE)
+    LEFT JOIN (
+        SELECT
+            d.feature_id,
+            array_agg(d.council_district_id) AS council_districts
+        FROM
+            features_council_districts d
+        GROUP BY
+            d.feature_id) districts ON districts.feature_id = feature_signals.id
```

Comment:
we're bringing each feature's council districts into an array of district IDs. i am pretty worried about the performance of this join! i think it might be running a full table scan on `features_council_districts`.

i am open to trying to optimizing it!

Reply:
hmmm not sure if I can offer more than questions here. It looks like having feature council districts in this view would impact what we could bring into `project_list_view` and `project_geography`. I think I see a workaround for `project_geography` but would this affect our ability to get council districts into the project list view? I might be on the wrong track.

Reply:
you're right, the hasura-based approach does not solve for the project list view, and realistically i think we're just going to need this column on this this view. i was browsing some SO posts and does seem like Postgres is pretty good at optimizing this kind of join on it's own. i think it's safe to ship this and revisit it later.